.github/CODEOWNERS

```diff
@@ -1,42 +1,42 @@
 # Default owner for all pull requests
-*    @terraform-providers/aws-provider
+*    @hashicorp/aws-provider
```

Comment:
FYI in the `hashicorp` Organization, we are `hashicorp/terraform-aws`: https://github.com/orgs/hashicorp/teams/terraform-aws